examples/tts/conf/fastpitch_align.yaml

```diff
@@ -0,0 +1,183 @@
+name: "FastPitch"
+sample_rate: 44100
```

Comment:
All our released models in TTS uses `sample_rate=22050`. Do you plan to release something with 44100?